src/main.cpp.orig

```diff
@@ -0,0 +1,52 @@
+#include <Arduino.h>
```

Comment:
Forgot to delete?

Reply:
sorry, shouldn't be added. missing in `.gitignore`, will fix it.